### DISCUSSION TITLE
Remove html comments for build

### DISCUSSION THREAD

**Discussion by memic84** [nuxt/nuxt#24287] [Questions]
Is is possible to remove the HTML comments during build in Vite?

**Comment by Youhan** [comment]
Try this in your nuxt config: 
```
'vite': {
    esbuild: {
      legalComments: 'none',
    },
    build: {
      terserOptions: {
        format: {
          comments: false,
        },
      },
    },
  },
```

**Reply by oooFreaKooo** [reply]
> Try this in your nuxt config:
> 
> ```
> 'vite': {
>     esbuild: {
>       legalComments: 'none',
>     },
>     build: {
>       terserOptions: {
>         format: {
>           comments: false,
>         },
>       },
>     },
>   },
> ```

Why isnt this default? I dont understand...
But thanks a lot man, cheers!
